app/raid.js

```diff
@@ -187,7 +185,15 @@ class Raid {
 	}
 
 	setMessage(channel, member, raid_id, message) {
-		this.getRaid(channel, member, raid_id).message = message;
+		const raid = this.getRaid(channel, member, raid_id),
+			current_message = raid.message;
+
+		if (current_message) {
+			current_message.delete();
+		}
+
+		message.pin()
+			.then(msg => raid.message = message);
```

Comment:
Is this pinning the message :P I guess it doesn't really matter if it's constantly pinning and unpinning things, other than the fact that it keeps spamming the channel about pinning things lol